lib/rack/lint.rb

```diff
@@ -278,11 +278,17 @@ def check_env(env)
       }
 
       ## The CGI keys (named without a period) must have String values.
+      ## If the string values contain non-ASCII characters, they should
+      ## use a ASCII-8BIT encoding.
       env.each { |key, value|
         next  if key.include? "."   # Skip extensions
         assert("env variable #{key} has non-string value #{value.inspect}") {
           value.kind_of? String
         }
+        next if value.encoding.to_s == "ASCII-8BIT"
```

Comment:
How about avoid to_s?

```suggestion
        next if value.encoding == Encoding::ASCII_8BIT
```

Reply:
That should be fine.  I figured this is Rack::Lint, micro optimization is not really necessary, but there's no harm in doing so.